csrc/driver_api.h

```diff
@@ -22,7 +22,18 @@ namespace nvfuser {
 // List of driver APIs that you want the magic to happen.
 DECLARE_DRIVER_API_WRAPPER(cuGetErrorName);
 DECLARE_DRIVER_API_WRAPPER(cuGetErrorString);
+DECLARE_DRIVER_API_WRAPPER(cuModuleLoadDataEx);
```

Comment:
nitpick on macro. `DECLARE_DRIVER_API_WRAPER` and `DEFINE_DRIVER_API_WRAPPER` seems to always show up in pairs. Maybe we can put them together as a macro and call wrapper on top?

something like
```
#define ALL_DRIVER_API_WRAPPER(fn) \
  fn(cuGetErrorName)                                  \
  fn(...)

// .h
ALL_DRIVER_API_WRAPPER(DECLARE_DRIVER_API_WRAPPER)
// .cpp
ALL_DRIVER_API_WRAPPER(DEFINE_DRIVER_API_WRAPPER)
```

Reply:
Unfortunately you can not use `#if (CUDA_VERSION >= 12000)` inside `ALL_DRIVER_API_WRAPPER`...

Reply:
We can put the #if outside.

Something like these: https://github.com/pytorch/pytorch/blob/a565f1bee68478c0237836d5e593601e9194083b/aten/src/ATen/cuda/nvrtc_stub/ATenNVRTC.h#L62-L70

Reply:
Good point! Let's include this into whatever next PR is for this topic.